stdlib/sqlite3/dbapi2.pyi

```diff
@@ -128,6 +129,18 @@ class _AggregateProtocol(Protocol):
     def step(self, value: int) -> None: ...
     def finalize(self) -> int: ...
 
+class _SingleParamWindowAggregateClass(Protocol):
+    def step(self, __param: Any) -> None: ...
+    def inverse(self, __param: Any) -> None: ...
```

Comment:
As an implementation could return a value, but users should ignore it:

```suggestion
    def step(self, __param: Any) -> object: ...
    def inverse(self, __param: Any) -> object: ...
```

Reply:
The same for `_WindowAggretateClass`.

Reply:
Done